examples/guestbook/README.md

```diff
@@ -205,45 +255,38 @@ redis-master                           master                  redis
+Once the replication controller is up, you can list the pods in the cluster, to verify that the master and slaves are running.  You should see a list that includes something like the following:
 
 ```shell
 $ kubectl get pods
-POD                                          IP                  CONTAINER(S)            IMAGE(S)                                 HOST                                                              LABELS                                                     STATUS
-redis-master-controller-gb50a                10.244.3.7          master                  redis                                    kubernetes-minion-7agi.c.hazel-mote-834.internal/104.154.54.203   name=redis-master                                          Running
-redis-slave-controller-182tv                 10.244.3.6          slave                   kubernetes/redis-slave:v2                kubernetes-minion-7agi.c.hazel-mote-834.internal/104.154.54.203   name=redis-slave                                           Running
-redis-slave-controller-zwk1b                 10.244.2.8          slave                   kubernetes/redis-slave:v2                kubernetes-minion-3vxa.c.hazel-mote-834.internal/104.154.54.6     name=redis-slave                                           Running
+NAME                                           READY     REASON    RESTARTS   AGE
+...
+redis-master-dz33o                             1/1       Running   0          2h
+redis-slave-35mer                              1/1       Running   0          2h
+redis-slave-iqkhy                              1/1       Running   0          2h
 ```
 
-You will see a single redis master pod and two redis slave pods.
+You should see a single redis master pod and two redis slave pods.  As mentioned above, you can get more information about any pod with: `kubectl describe pods/<pod_name>`.
 
 ### Step Four: Create the redis slave service
 
-Just like the master, we want to have a service to proxy connections to the redis slaves. In this case, in addition to discovery, the slave service provides transparent load balancing to web app clients.
-
-The service specification for the slaves is in `examples/guestbook/redis-slave-service.json`:
-
-```js
-{
-   "kind":"Service",
-   "apiVersion":"v1",
-   "metadata":{
-      "name":"redis-slave",
-      "labels":{
-         "name":"redis-slave"
-      }
-   },
-   "spec":{
-      "ports": [
-        {
-          "port":6379
-        }
-      ],
-      "selector":{
-         "name":"redis-slave"
-      }
-   }
-}
+Just like the master, we want to have a service to proxy connections to the redis slaves. In this case, in addition to discovery, the slave service will provide transparent load balancing to web app clients.
+
+The service specification for the slaves is in `examples/guestbook/redis-slave-service.yaml`:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    name: redis-slave
+  name: redis-slave
```

Comment:
@amygdala, this one is not fixed. I left comments for all of them so that I can see if all of them have become "caesarxuchao commented on an outdated diff" :)